src/main/java/twilightforest/TFFeature.java

```diff
@@ -378,6 +378,15 @@ public static TFFeature generateFeatureForOldMapGen(int chunkX, int chunkZ, Worl
     }
 
     public static TFFeature generateFeatureFor1Point7(int chunkX, int chunkZ, World world) {
```

Comment:
It appears this function isn't only used when generating a feature (as its name would suggest) but also when looking up which feature should exist in a given region.

Changing the configurated change post-worldgen affects the functionality of functionality that check against existing features such as the magic map.

Is this intended? If so you might want to document the caveat that this config isn't meant to be modified on an existing world.
![image](https://github.com/GTNewHorizons/twilightforest/assets/5321549/b9b80c4c-4344-4d5c-a77f-179ed29fcf17)

Reply:
yes, definitely need a warning, I was worried that this would happen